rllib/algorithms/algorithm_config.py

```diff
@@ -2426,6 +2474,18 @@ def validate_train_batch_size_vs_rollout_fragment_length(self) -> None:
                     f"{suggested_rollout_fragment_length}."
                 )
 
+    @classmethod
+    def get_rl_module_class(cls, framework_str: str) -> Union[Type["RLModule"], str]:
```

Comment:
`ppo` for example would override this method in its own PPOConfig and return `PPOTorchRLModule`.

Reply:
I think you want this to be ``@staticmethod``?
``cls`` should not be useful in any case.

Reply:
Not sure about this. Seems overkill.

Better:
* PPOConfig (or the config for the PPO version that uses RLModules) sets its `rl_module_class` property to `PPORLModule` as default value in its contructor.
* `PPOConfig.rl_module()` (or the config for the PPO version that uses RLModules) immediately verifies, whether a possible, user provided custom RLModule is a subclass of `PPORLModule`.
* Then we just need to read the property `rl_module_class`, instead of having to call this method.

Reply:
I never learned when I should use staticmethod vs. classmethod :) staticmethods in my opinion are useless in python. You may as well add them outside of the scope of a class. :) Anyway switching it to an instance method to allow more complicated rl_module routings based on maybe `exploration` keywords.